### PR TITLE
Mainify some slow euler solutions

### PR DESCRIPTION
**Pull request by mnil2**
While experimenting with different timeout values, two euler solutions were slow enough on travis to rival some of the meatier imports. I have mainified them to be safe that they don't fail to import if travis is feeling slow or something.

#21

### CODE REVIEW

File: thealgorithms/project_euler/problem_36/sol1.py [hunk 1]
```diff
@@ -8,23 +8,27 @@
...
-	xrange = range	#Python 3
+def main():
+    try:
+	    xrange		#Python 2
+    except NameError:
+	    xrange = range	#Python 3
 
-def is_palindrome(n):
-	n = str(n)
+    def is_palindrome(n):
```
Comment (rouwaydhanna) [inline thealgorithms/project_euler/problem_36/sol1.py]: Would be nice if this function was declared outside of main

File: thealgorithms/project_euler/problem_36/sol1.py [hunk 2]
```diff
@@ -8,23 +8,27 @@
 
 (Please note that the palindromic number, in either base, may not include leading zeros.)
 '''
-try:
-	xrange		#Python 2
-except NameError:
-	xrange = range	#Python 3
+def main():
+    try:
+	    xrange		#Python 2
```
Comment (rouwaydhanna) [inline thealgorithms/project_euler/problem_36/sol1.py]: Double tab here?

**Review by rouwaydhanna — CHANGES_REQUESTED** [review]
Small stuff, just declare functions outside main and there seems to be one extra tab in if, while and try-catch bodies, see example comment.

File: thealgorithms/project_euler/problem_36/sol1.py (hunk 1 above)
Comment (mnil2) [reply thealgorithms/project_euler/problem_36/sol1.py]: True I didnt see this one

File: thealgorithms/project_euler/problem_36/sol1.py (hunk 2 above)
Comment (mnil2) [reply thealgorithms/project_euler/problem_36/sol1.py]: No but i was using spaces when the original code had tabs. I changed my spaces into tabs, should look consistent now

**Review by rouwaydhanna — APPROVED** [review]
Nice 👍

**Pull request merged**